CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## 0.21.0 (Unreleased)
 
+## 0.20.3 (September 23, 2020)
+
+BUG FIXES:
+
+* resource/fastly_service_v1/bigquerylogging: Ensure BigQuery logging `email`, `secret_key` fields are required and not optional. ([#319](https://github.com/terraform-providers/terraform-provider-fastly/pull/319))
```

Comment:
```suggestion
* resource/fastly_service_v1/bigquerylogging: Ensure BigQuery logging `email`, `secret_key` fields are required and not optional. ([#319](https://github.com/fastly/terraform-provider-fastly/pull/319))
```

Reply:
Should this be the new repo?

Reply:
Good catch, have cleaned up the rest in #321